docs/get-started/detect-people.md

```diff
@@ -97,9 +97,7 @@ Create the service.
 
 In the resulting ML model service configuration pane, ensure that **Deploy model on machine** is selected for the **Deployment** field.
 
-Click on **Select models** and select the `ml-models-scuttle:people` model from the registry (the **people** model by **ml-models-scuttle**) to deploy an object detection TFLite model that has been trained to be able to detect a person.
-
-Click the **Save** button in the top right corner of the page to save your configuration and deploy an object detection TFLite model that has been trained to be able to detect a person.
```

Comment:
I do think we should still tell people to save it though. here I meant to repeat to drill it in ("tell show tell")

Reply:
why though? they will be saving immediately again right after